composer.json

```diff
@@ -18,6 +18,7 @@
         "php": ">=7.4",
         "composer-plugin-api": "^2",
         "acquia/drupal-environment-detector": "^1.5.3",
+        "acquia/memcache-settings": "^1.2",
```

Comment:
@rajeshreeputra This is not required. Let's keep it, similar to acquia/blt.

Reply:
sure, as discussed, removed.